utils/parts.stanza

```diff
@@ -495,9 +495,21 @@ defn NoConnectCode (json: JObject) -> NoConnectCode :
   NoConnectCode(PinByTypeCode(json["pin"] as JObject))
 
 public defmethod to-jitx (part: Part) -> Instantiable :
-  to-jitx $ component(part)
+  to-jitx(component(part), sellers(part))
 
 public defn to-jitx (c: ComponentCode) -> Instantiable :
+  to-jitx(c, false)
+
+; Set sellers as a property so they show up in the component card.
+public defn set-sellers-as-property (sellers: Tuple<Seller>|False) :
+  match(sellers:Tuple<Seller>) :
+    inside pcb-component :
+      var sellers-str = ""
+      for seller in sellers do :
+        sellers-str = append-all([sellers-str, "\n", to-string(seller)])
+      set-property(self, `sellers, sellers-str)
```

Comment:
This should be
```
; Set sellers as a property so they show up in the component card.
 public defn set-sellers-as-property (sellers: Tuple<Seller>|False) :
  match(sellers:Tuple<Seller>) :
    property(self.sellers) = sellers
```

Reply:
The stanza way to write your original code would have been `to-string("%n" % [sellers])` or `string-join(sellers, "\n")`

Reply:
Thanks @PhilippeFerreiraDeSousa!

Have updated with this (and above changes in jitx-client), and re-tested.